Dockerfile

```diff
@@ -145,7 +146,14 @@ RUN add-apt-repository ppa:ondrej/php \
     php7.4-tidy \
     php7.4-xml \
     php7.4-zip \
-    php7.4-zmq
+    php7.4-zmq \
+  # Perform a fake `composer update` to warm ~dependabot/.composer/cache with historic data
```

Comment:
Concern: this `RUN` just did an `apt-get`, so it must be `USER root`.
Are we sure this isn't warming `~root/.composer/cache`? That's less useful.
If we want that, can we adjust the comment?

We may want to split L151-156 out to another `RUN`, and wrap in `USER`s.

Reply:
Oh good catch, I think you're right, will confirm.

Reply:
Yep, fixed this and deleted the downloaded package files